plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/maven2/Maven2GroupFacet.java

```diff
@@ -84,12 +88,28 @@ protected void doInit(final Configuration configuration) throws Exception {
     this.mavenFacet = facet(MavenFacet.class);
   }
 
+  @Override
+  protected void doConfigure(final Configuration configuration) throws Exception {
+    super.doConfigure(configuration);
+    if (cacheController == null) {
+      cacheController = new CacheController(1440, null);
```

Comment:
Can we add `metadataMaxAge` or `contentMaxAge` as a configuration element for `Maven2GroupFacet` rather than rely on a hard-coded value?